manifests/l/Logitech/CameraSettings/2.12.20/Logitech.CameraSettings.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS0.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Logitech.CameraSettings
+PackageVersion: 2.12.20
```

```suggestion
PackageVersion: 2.12.20.0
```
Please update file name and path to match this change.